bin/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+REGION=us-east-2
+ENVIRONMENT=${1:-dev}
+CLUSTER_NAME=$ENVIRONMENT-cluster0
```

Comment:
nit: currently, we only ever use cluster0, but maybe we could have the cluster index be a script arg w/ a default in the same way `ENVIRONMENT` is?

Reply:
good suggestion, done